### PR TITLE
chore: Migrate widgets from separate repo to monorepo

### PR DESCRIPTION
**Pull request by jmrossy**
### Description

Copies the [widget lib](https://github.com/hyperlane-xyz/hyperlane-widgets) over so that it can use the same review and release process as other packages.
No code changes, only some minor tooling config changes to work in monorepo.

### Related issues

Fixes https://github.com/hyperlane-xyz/issues/issues/1111

### Backward compatibility

Yes

### Testing

Tested in storybook

### CODE REVIEW

File: typescript/widgets/.eslintignore
Comment (nbayindirli) [inline typescript/widgets/.eslintignore:1]: nit: no newline eof

File: typescript/widgets/.gitignore
Comment (nbayindirli) [inline typescript/widgets/.gitignore:1]: nit: no newline eof

File: typescript/widgets/.storybook/preview-head.html
Comment (nbayindirli) [inline typescript/widgets/.storybook/preview-head.html:1]: nit: no newline eof

File: typescript/widgets/tsconfig.json
Comment (nbayindirli) [inline typescript/widgets/tsconfig.json:1]: nit: no newline eof

File: typescript/helloworld/LICENSE.md
Comment (nbayindirli) [inline typescript/helloworld/LICENSE.md:1]: why remove?

**Review by nbayindirli — APPROVED** [review]

File: typescript/helloworld/LICENSE.md
Comment (jmrossy) [reply typescript/helloworld/LICENSE.md:1]: It was redundant with the one in the monorepo root. A vestige of when this package was in it's own repo (like widgets was)

**Pull request merged**